packages/components/statistic/src/Statistic.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="ix-statistic">
+    <div v-if="title || $slots.title" class="ix-statistic-title">
+      <slot name="title">{{ title }}</slot>
+    </div>
+    <div class="ix-statistic-content">
+      <span v-if="prefix || $slots.prefix" class="ix-statistic-content-prefix">
+        <slot name="prefix">{{ prefix }}</slot>
+      </span>
+      <span class="ix-statistic-content-value">
+        <slot>
+          <span class="ix-statistic-content-value-int">{{ formatedValue.int || formatedValue.value }}</span>
+          <span v-if="formatedValue.decimal" class="ix-statistic-content-value-decimal">
+            .{{ formatedValue.decimal }}
```

Comment:
```suggestion
            {{ formatedValue.decimal }}
```
在format 的时候，就把分隔符包含在 decimal 内。便于用户更好的自定义，
另外，我看 ng 的代码里，国际化处理的时候，分隔符不一定是小数点，我们后续增加国际化需求的时候也可能需要考虑。